make_queue/templates/make_queue/reservation_machines.html

```diff
@@ -27,6 +27,10 @@
             <a href="{% url "find_free_slot" %}" class="ui primary button">
                 {% trans "Find free reservation slots" %}
             </a>
+            <a href="{% url "SLA_request" %}" class="ui primary button">
```

Comment:
Two things:
- The SLA printer is currently out of order, so it does not make sense to open up for requests. Maybe it should be possible to toggle the visibility of this button?
- As we add more functionality to the reservation system, we end up with more and more blue buttons here. It is ok for now with only three buttons, but I think we maybe should rethink the design here.